pkg/clients/aws/aws.go

```diff
@@ -0,0 +1,148 @@
+package aws
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	awscredsv2 "github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	awscredsv1 "github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/openshift/osd-network-verifier/pkg/helpers"
+)
+
+// Client represents an AWS Client
+type Client struct {
```

Comment:
unit tests are still getting created via
```go
	cli := Client{
		ec2Client: FakeEC2Cli,
		logger:    &logging.GlogLogger{},
	}
```
so they're failing on my local

Reply:
Yes unit tests need to be fully rewritten. That will be my next step as well as fixing the gomocs like you mention above